src/components/ContentEditable.tsx

```diff
@@ -1,5 +1,10 @@
 import React, { useEffect, useRef } from 'react'
 import { useSelector } from 'react-redux'
+import * as selection from '../device/selection'
+import exportContext from '../selectors/exportContext'
+import simplifyPath from '../selectors/simplifyPath'
+import store from '../stores/app'
```

Comment:
We generally want to avoid importing the global store. Instead, stick to `useSelector` so that we can respect the normal flow of data in a React-Redux app. Otherwise it can lead to stale state.